open-api/rest-catalog-open-api.yaml

```diff
@@ -3103,6 +3103,82 @@ components:
         uuid:
           type: string
 
+    ADLSCredentials:
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/Credentials'
+      required:
+        - type
+        - sas-token
+        - expires-at-ms
+      properties:
+        type:
+          type: string
+          enum: [ "adls" ]
+        sas-token:
+          type: string
+        expires-at-ms:
+          type: integer
+          format: int64
+          description: The epoch millis at which the given token expires
```

Comment:
nit: sorry, but "the epoch" is still not precise :) Would you mind mentioning a specific point in time (as in javadoc of `System.currentTimeMillis()`?... i.e. `Milliseconds since January 1, 1970 UTC`?

Reply:
updated